### PR TITLE
Notes api

### PR DESCRIPTION
**Pull request by waqasidrees15**

### CODE REVIEW

File: backend/app/database/config.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from sqlalchemy.orm import sessionmaker
 
 
-DATABASE_URL = "postgresql://postgres:waqas123HFD@localhost/Notes_db"
+DATABASE_URL = "postgresql://postgres:waqas123HFD@localhost/notes"
```
Comment (muiz6) [inline backend/app/database/config.py]: Database url should be in .env file. Explore python-dotenv package.

File: backend/app/routers/routes.py [hunk 2]
```diff
@@ -1,54 +1,73 @@
...
+@router.delete("/notes/delete")
 async def delete_notes(request: RequestNotes, db: Session = Depends(get_db)):
     crud.remove_notes(db, notes_id=request.parameter.id)
     return Response(status="Ok", code="200", message="Success delete data").dict(
         exclude_none=True
     )
+
+
+@router.post('/login/facebook')
+async def create_client_service(access_token: str, db: Session = Depends(get_db)):
```
Comment (muiz6) [inline backend/app/routers/routes.py]: access token should be in form data

File: backend/app/routers/routes.py [hunk 3]
```diff
@@ -1,54 +1,73 @@
...
-@router.delete("/delete")
+@router.delete("/notes/delete")
 async def delete_notes(request: RequestNotes, db: Session = Depends(get_db)):
     crud.remove_notes(db, notes_id=request.parameter.id)
     return Response(status="Ok", code="200", message="Success delete data").dict(
         exclude_none=True
     )
+
+
+@router.post('/login/facebook')
```
Comment (muiz6) [inline backend/app/routers/routes.py]: swagger docs should show return type for all routes

**Review by muiz6 — CHANGES_REQUESTED** [review]

File: backend/app/database/config.py (hunk 1 above)
Comment (waqasidrees07) [reply backend/app/database/config.py]: Issue resolved

File: backend/app/routers/routes.py (hunk 2 above)
Comment (waqasidrees07) [reply backend/app/routers/routes.py]: solved

File: backend/app/routers/routes.py (hunk 3 above)
Comment (waqasidrees07) [reply backend/app/routers/routes.py]: solved

**Pull request merged**